plans/rpminspect-centos.fmf

```diff
@@ -0,0 +1,22 @@
+summary:
+    Check rpm files with rpminspect
+discover:
+    how: shell
+    tests:
+      - name: rpminspect SRPM
+        test: rpminspect-centos -E metadata -v -t VERIFY --profile=centos-stream-9-devel /tmp/*.src.rpm
+      - name: rpminspect binary RPMs
+        test: for rpm in /var/share/test-artifacts/*.rpm; do rpminspect-centos -E metadata -v -t VERIFY --profile=centos-stream-9-devel $rpm; done
+prepare:
+  - name: packages
+    how: install
+    package:
+    - rpminspect
+    - rpminspect-data-centos
+  - how: shell
+    script: cd /tmp && curl -O ${PACKIT_SRPM_URL}
```

Comment:
```suggestion
    script: dnf download ${PACKIT_COPR_RPMS} --destdir=$TMT_PLAN_DATA
```
A simpler one-liner, and then in `tests`, change the path accordingly. Should be more robust against future environment changes.

Reply:
Good idea!

Reply:
Maybe, the only downside might be the lack of DNF on some systems.